crisischeckin/CrisisCheckinMobile/CrisisCheckinMobile/ViewModels/ProfileViewModel.cs

```diff
@@ -0,0 +1,187 @@
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace CrisisCheckinMobile.ViewModels
+{
+    public class ProfileViewModel : INotifyPropertyChanged
+    {
+        private INavigation navigation;
+
+        public ProfileViewModel(INavigation navigation)
+        {
+            this.navigation = navigation;
+        }
+
+
+        public ICommand GoToMyContacts
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO Change this to Contacts Page when ready.
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToMyRequests
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToMyInfo
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToViewDisasterData
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToViewDisasterResources
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand AddResources
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand AddContacts
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand ReportData
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand CheckOutOfDisaster
+        {
+            get
+            {
+                return new Command(() =>
+                {
+                    this.IsCheckedInVisible = true;
```

Comment:
This delegate changes the backing fields, not through the property setter. I believe that means this command will not raise the PropertyChanged event, and the UI will be out of sync. Let's make sure we test that interactively before merging.